### ISSUE TITLE
portal_angular_dependencies: rename task and output file

### ISSUE THREAD

**Issue by x1B** [LaxarJS/grunt-laxar#29]
After #28, the task name has become misleading.
This task and its output should be renamed to `widget_dependencies`.

**Comment by jpommerening** [comment]
The same thing came up just today, good to see this here :)
/cc @mkemmann

**Comment by x1B** [comment]
Scratch that, it's _laxar-application-dependencies_, not _widget_dependencies_.

**Comment by jpommerening** [comment]
:+1:

**Comment by x1B** [comment]
### Breaking Change

The `grunt-laxar` task `portal_angular_dependencies` has been renamed to `laxar_application_dependencies`. The reasonsl: the term _portal_ is somewhat obscure as it has been removed from LaxarJS core and is now called _runtime_. Also, the collected dependencies do not actually belong to the runtime, but of the _application widgets_ and _controls._ And finally, the task collects and widget AMD dependencies, not just AngularJS, making the old name very misleading.


#### Upgrade Information

When upgrading to LaxarJS `v1.0.0`, the task reference in the _Gruntfile_ needs to be changed, for example like this:

```patch
@@ ... @@
-      portal_angular_dependencies: {
+      laxar_application_dependencies: {
          default: {
             options: {},
-            dest: 'var/static/portal_angular_dependencies.js',
+            dest: 'var/static/laxar_application_dependencies.js',
             src: [ 'application/flow/*.json' ]
          }
       },

@@ ... @@
-   grunt.registerTask( 'build', [ 'directory_tree', 'portal_angular_dependencies' ] );
+   grunt.registerTask( 'build', [ 'directory_tree', 'laxar_application_dependencies' ] );
```

To match the renamed task and dependencies file, the require-configuration should be adjusted:

```patch
@@ ... @@
-      portal_angular_dependencies: '../var/static/portal_angular_dependencies',
+      'laxar-application-dependencies': '../var/static/laxar_application_dependencies',
 ```

Same goes for the application initialization code, which should look similar to this:

```js
require( [
   'laxar',
   'laxar-application-dependencies'
], function( ax, applicationDependencies ) {
   'use strict';

   ax.bootstrap( applicationDependencies );

} );
```


### Availabilty

Implemented on `master` (`v1.0.0`).
